src/tiledb/cloud/bioimg/ingestion.py

```diff
@@ -11,6 +11,7 @@
 DEFAULT_IMG_NAME = "3.9-imaging-dev"
 DEFAULT_DAG_NAME = "bioimg-ingestion"
 _SUPPORTED_EXTENSIONS = (".tiff", ".tif", ".svs")
+_SUPPORTED_CONVERTERS = ("tiff", "zarr", "svs")
```

Comment:
Shouldn't `osd` be here?
Why allow `svs` here, it's not a specific converter type...